wpimath/src/main/java/edu/wpi/first/math/estimator/MerweScaledSigmaPoints.java

```diff
@@ -69,8 +69,10 @@ public int getNumSigmas() {
   }
 
   /**
-   * Computes the sigma points for an unscented Kalman filter given the mean (x) and covariance(P)
+   * Computes the sigma points for an unscented Kalman filter given the mean (x) and square root covariance(s)
    * of the filter.
+   * 
+   * (Eq. 17)
```

Comment:
Put the equation references in the code itself. The user doesn't care about those details, so they don't need to be in the API docs.